### PR TITLE
fix/1inch convert

### LINKED ISSUE #331: dApp crashes when trying to convert specific ERC-20 tokens to sUSD on "Convert" page
**Description:**
dApp crashes when trying to convert specific ERC-20 tokens to sUSD on "Convert" page.

**Prerequisites:**
- specific ERC-20 token on the wallet (UNI or RAZOR for example)

**Reproduction steps:**
1. Go to Kwenta "Convert" page (https://dev.kwenta.io/dashboard/convert)
2. Connect your wallet
3. Select any token (mentioned above, for example "UNI") inside "From" drop-down
4. Type any value inside "From" input field
5. Click on "Approve" button

**Current behavior:**
- dApp crashes after hitting "Approve" button for "UNI" token

**Expected behavior:**
- user should be able to convert all assets with success

**Screenshots:**

Screenshot 2021-04-12 at 18 19 20

**Desktop (please complete the following information):**
 - OS: macOS Big Sur 11.2.2
 - Browser: Chrome 89.0.4389.95

### PR DESCRIPTION
**Pull request by evgenyboxer**
fixes #331 and #337

### CODE REVIEW

File: sections/dashboard/CurrencyConvertCard/SlippageSelector.tsx [hunk 1]
```diff
@@ -45,11 +50,19 @@ export const SlippageSelector: FC<SlippageSelectorProps> = () => {
...
+										setCustomSlippage('');
+										setSlippage(DEFAULT_SLIPPAGE);
+									} else if (numValue === 0) {
+										setCustomSlippage(value);
+										setSlippage(MIN_SLIPPAGE);
+									} else {
+										const clampedNum = clamp(numValue, MIN_SLIPPAGE, MAX_SLIPPAGE);
+										setSlippage(clampedNum);
+										setCustomSlippage(`${clampedNum}`);
 									}
```
Comment (evgenyboxer) [inline sections/dashboard/CurrencyConvertCard/SlippageSelector.tsx:65]: The logic here makes the input value be between `MIN_SLIPPAGE` and `MAX_SLIPPAGE` while also making the input clearable.

File: sections/dashboard/CurrencyConvertCard/SlippageSelector.tsx [hunk 2]
```diff
@@ -45,11 +50,19 @@ export const SlippageSelector: FC<SlippageSelectorProps> = () => {
...
-										const numValue = Number(value);
-										setSlippage(numValue >= 0 ? numValue : DEFAULT_SLIPPAGE);
+									const numValue = Number(value);
+
+									if (value === '') {
+										setCustomSlippage('');
+										setSlippage(DEFAULT_SLIPPAGE);
+									} else if (numValue === 0) {
+										setCustomSlippage(value);
+										setSlippage(MIN_SLIPPAGE);
```
Comment (evgenyboxer) [inline sections/dashboard/CurrencyConvertCard/SlippageSelector.tsx:60]: note - altho the custom value here is "0", I don't really think the user will ever want "0" slippage, so I set a min slippage of 0.01 (0 is valid, but will most likely result in a failed tx)

**Review by JChiaramonte7 — APPROVED** [review]
Looks good. Only found a small UI issue. 

image

image

Extra space under the synth img messing with height of div. I think `vertical-align: middle;` on Currency.Icon should fix it (or at least it did in chrome debugger)

https://stackoverflow.com/questions/10844205/html-5-strange-img-always-adds-3px-margin-at-bottom

File: sections/dashboard/CurrencyConvertCard/SlippageSelector.tsx (hunk 2 above)
Comment (andytcf) [reply sections/dashboard/CurrencyConvertCard/SlippageSelector.tsx:60]: it might be better tho, to actually disable 0 slippage input

**Review by andytcf — APPROVED** [review]

**Comment by evgenyboxer** [comment]
@JChiaramonte7 @andytcf I'll open a new PR with the recommendations so that @drptbl can test keep testing.

**Pull request merged**